### PR TITLE
update startup

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

**Review by Cheephot — APPROVED** [review]

File: src/SearchService/appsettings.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "CheckUserIsAdminEndpoint": "UserService_GetUserAdminStatus",
     "SearchUsersEndpoint": "UserService_SearchUsers",
     // CompanyService
```
Comment (MariyaSin) [inline src/SearchService/appsettings.json]: DepartmentService

**Review by MariyaSin — APPROVED** [review]

File: src/SearchService/appsettings.json (hunk 1 above)
Comment (BelehovEgor) [reply src/SearchService/appsettings.json]: fixed

File: src/SearchService/Startup.cs [hunk 2]
```diff
@@ -12,126 +14,153 @@
...
+        .GetSection(BaseServiceInfoConfig.SectionName)
+        .Get<BaseServiceInfoConfig>();
 
-            app.UseExceptionsHandler(loggerFactory);
+      _rabbitMqConfig = Configuration
+        .GetSection(BaseRabbitMqConfig.SectionName)
+        .Get<RabbitMqConfig>();
 
-            app.UseApiInformation();
+      Version = "1.0.0";
```
Comment (VLiamin) [inline src/SearchService/Startup.cs:93]: May be 1.0.0.0?

**Review by VLiamin — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 3]
```diff
@@ -10,178 +14,190 @@
...
-                {
-                    projects.AddRange(
-                        response.Message.Body.Entities.Select(
-                            p => new SearchResultInfo
-                            {
-                                Type = SearchResultObjectType.Project,
-                                Id = p.Id,
-                                Value = p.Value
-                            }).ToList());
+        errors.Add(errorMessage);
```
Comment (ndoriomedov) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs]: Can move it and line 62 to line 65

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 4]
```diff
@@ -10,178 +14,190 @@
...
+    private async Task<List<SearchResultInfo>> SearchProjectsAsync(string text, List<string> errors)
     {
-        private IRequestClient<ISearchProjectsRequest> _rcProjects;
-        private IRequestClient<ISearchUsersRequest> _rcUsers;
-        private IRequestClient<ISearchDepartmentsRequest> _rcDepartments;
-        private ILogger<SearchCommand> _logger;
+      List<SearchResultInfo> projects = new();
 
-        #region requests
+      string errorMessage = "Cannot search projects. Please try again later.";
```
Comment (ndoriomedov) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs:33]: Do we need variable for this string?

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 5]
```diff
@@ -10,178 +14,190 @@
...
+        }
 
-            string errorMessage = "Cannot search departments. Please try again later.";
+        _logger.LogWarning(errorMessage);
 
-            try
-            {
-                var response = _rcDepartments.GetResponse<IOperationResult<ISearchResponse>>(
-                    ISearchDepartmentsRequest.CreateObj(text)).Result;
+        errors.Add(errorMessage);
```
Comment (ndoriomedov) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs]: Same

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs [hunk 6]
```diff
@@ -10,178 +14,190 @@
...
+          return users;
         }
 
-        private List<SearchResultInfo> SearchUsers(string text, List<string> errors)
-        {
-            List<SearchResultInfo> users = new();
+        _logger.LogWarning(errorMessage);
 
-            string errorMessage = "Cannot search users. Please try again later.";
+        errors.Add(errorMessage);
```
Comment (ndoriomedov) [inline src/SearchService.Bussines/Commands/Search/SearchCommand.cs]: Same

**Review by ndoriomedov — APPROVED** [review]

**Review by asyagraf — APPROVED** [review]

File: src/SearchService.Bussines/Commands/Search/SearchCommand.cs (hunk 4 above)
Comment (MariyaSin) [reply src/SearchService.Bussines/Commands/Search/SearchCommand.cs:33]: yes)

File: src/SearchService/Startup.cs (hunk 2 above)
Comment (MariyaSin) [reply src/SearchService/Startup.cs:93]: we create in when change spec

**Pull request merged**